README.md

```diff
@@ -57,41 +60,103 @@ If you are interested in becoming a member of the team check out the **[Pixelogi
 We found [this guide](https://www.digitalocean.com/community/tutorials/understanding-the-gopath) pretty helpful in understanding the Golang file structure and how it should be setup. We have tweaked this file structure to fit our Firebase Function setup.
 
 1. Make sure to [install Golang](https://golang.org/dl/) we are using v1.13
-2. Find your `GOPATH`. ALWAYS located in this path (`$HOME/go`) unless put otherwise
-   1. Example GOPATH: `Users/YourComputerName/go`
-   2. Example GOPATH/BIN:`Users/YourComputerName/go/bin`
-3. Add GOPATH env variable
+1. Find your `GOPATH`. ALWAYS located in this path (`$HOME/go`) unless put otherwise
+    1. Example GOPATH: `Users/YourComputerName/go`
+    1. Example GOPATH/BIN:`Users/YourComputerName/go/bin`
+1. Add GOPATH env variable
 
-```
+```shell
 export GOPATH="$HOME/go"
 export GOBIN="$HOME/go/bin"
 export PATH=$PATH:$GOBIN:$GOPATH
 ```
 
 ### Golang and VSCode Extensions
 
-We have found some awesome workflows and tools to get us up and running with Golang. This repo includes a [`.vscode/settings.json`](.vscode/settings.json) & [`.vscode/extensions.json`](.vscode/extensions.json) which will allow you to automatically download the recommended extenstions and keeps your repo styling in sync with all the contributors. If you find more extenstions please join the [PixelogicDev Discord](https://discordapp.com/invite/8NFtvp5) and share with us so we can add it to the project!
+We have found some awesome workflows and tools to get us up and running with Golang. This repo includes a [`.vscode/settings.json`](.vscode/settings.json) & [`.vscode/extensions.json`](.vscode/extensions.json) which will allow you to automatically download the recommended extensions and keeps your repo styling in sync with all the contributors. If you find more extensions please join the [PixelogicDev Discord](https://discordapp.com/invite/8NFtvp5) and share with us so we can add it to the project!
 
 ### Grüvee Backend File Path
 
 Make sure `gruveebackend` is in your GOPATH (This helps a lot. We promise.)
 
-1.  Clone repo from Github
-2.  Open up `GOPATH/src/github.com/`
-3.  Create folder called `pixelogicdev`
-4.  `cd pixelogicdev` and move/clone `gruveebackend` into this folder
+1. Clone repo from Github
+1. Open up `GOPATH/src/github.com/`
+1. Create folder called `pixelogicdev`
+1. `cd pixelogicdev` and move/clone `gruveebackend` into this folder
```

Comment:
[nit]

Numbering here as well!

Reply:
I always number with all 1. 's it is fully supported by markdown (which will number accordingly). And with this practice it allows you to reorder the list very easily.